pkg/apis/core/types_shoot.go

```diff
@@ -981,6 +983,39 @@ var (
 	DefaultWorkerMaxUnavailable = intstr.FromInt(0)
 )
 
+//////////////////////////////////////////////////////////////////////////////////////////////////
+// System components relevant types                                                             //
+//////////////////////////////////////////////////////////////////////////////////////////////////
+
+// SystemComponents contains the settings of system components in the control or data plane of the Shoot cluster.
+type SystemComponents struct {
+	// CoreDNS contains the settings of the Core DNS components running in the data plane of the Shoot cluster.
+	CoreDNS *CoreDNS `json:"coreDNS" protobuf:"bytes,1,opt,name=coreDNS"`
```

Comment:
Drop all `json` tags etc from internal APIs

Reply:
Removed the `json` tags from the internal APIs.